### ISSUE TITLE
Update outdated pods

### ISSUE THREAD

**Issue by elibud** [wordpress-mobile/WordPress-iOS#9102]
```- AFNetworking 3.1.0 -> 3.1.0 (latest version 3.2.0)
- Alamofire 4.7.0 -> 4.7.0 (latest version 4.7.1)
- Fabric 1.7.5 -> 1.7.6 (latest version 1.7.6)
- GTMSessionFetcher 1.1.14 -> 1.1.15 (latest version 1.1.15)
- lottie-ios 1.5.1 -> 1.5.1 (latest version 2.5.0)
- Nimble 7.0.3 -> 7.0.3 (latest version 7.1.0)
- Starscream 3.0.4 -> 3.0.4 (latest version 3.0.5)
```

**Comment by ScoutHarris** [comment]
@nheagy @elibud - I'm removing ZendeskSDK from this ticket and I'll do it separately. It seems ZD has completely changed how to add the SDK to a project with their latest version (which is now 2.0), so it's much more complicated than just updating it.

**Comment by diegoreymendez** [comment]
Ping @nheagy - Moving this out of 9.9 since it's frozen, but do feel free to bring it back if it'll be merged for 9.9.
